crates/bevy_ecs/src/query/access.rs

```diff
@@ -2,13 +2,14 @@ use crate::storage::SparseSetIndex;
 use fixedbitset::FixedBitSet;
 use std::marker::PhantomData;
 
-/// `Access` keeps track of read and write accesses to values within a collection.
+/// Tracks read and write access to specific elements in a collection.
 ///
-/// This is used for ensuring systems are executed soundly.
+/// System initialization and execution uses these to ensure soundness.
+/// See the [`is_compatible`](Access::is_compatible) and [`get_conflicts`](Access::get_conflicts) functions.
 #[derive(Debug, Eq, PartialEq, Clone)]
 pub struct Access<T: SparseSetIndex> {
     reads_all: bool,
-    /// A combined set of T read and write accesses.
+    writes_all: bool,
     reads_and_writes: FixedBitSet,
```

Comment:
`is_compatible` and `get_conflicts` seem wrong to me but honestly its extremely hard to tell because I have absolutely no idea what the difference is between `reads_and_writes` and `writes` like... all write access imply read access..? And where is the bitset for list of reads? :sweat_smile:

Reply:
Yes, write access implies read access, so `reads_and_writes` is bitset for the reads. 

There has never been a bitset for "read-only" access.

I just removed the comment because it was a private field and I thought my doc updates made it clearer.

Reply:
Resolving in favor of centralizing discussion under my latest comment.